Jenkinsfile.groovy

```diff
@@ -10,7 +10,7 @@ if (currentBuild.rawBuild.getCauses().toString().contains('BranchIndexingCause')
 }
 
 library(
-    identifier: "jsadmin_pipeline@master",
+    identifier: "jsadmin_pipeline@UITOOL-59/convert-pnpm-lock-to-npm",
```

Comment:
You already have a PR to switch that, right?

Reply:
yup https://github.com/coveo/react-vapor/pull/2085

Reply:
actually I merged it now that https://github.com/coveo/jsadmin_pipeline/pull/30 is merged